src/main/java/api/domain/cart/items/Skus.java

```diff
@@ -12,7 +12,7 @@
     @JsonProperty("product_id")
     private String productId;
     @JsonProperty("sku_id")
-    private Integer skuId;
+    private Integer sku_id;
```

Comment:
@GeluHai it should work as private Integer skuId; you already have @JsonProperty set with the right naming. You only need to generate getters and setters

Reply:
If I use private Integer skuId it doesn't work, I tried